examples/app/selection_histogram.py

```diff
@@ -59,22 +66,18 @@
 vzeros = np.zeros(len(vedges)-1)
 vmax = max(vhist)*1.1
 
-th = 42 # need to adjust for toolbar height, unfortunately
-pv = figure(toolbar_location=None, plot_width=200, plot_height=p.plot_height+th-10, x_range=(-vmax, vmax),
-            y_range=p.y_range, title=None, min_border=10, min_border_top=th)
+pv = figure(toolbar_location=None, plot_width=200, plot_height=p.plot_height, x_range=(-vmax, vmax),
+            y_range=p.y_range, min_border=10, y_axis_location="right")
 pv.ygrid.grid_line_color = None
-pv.xaxis.major_label_orientation = -3.14/2
+pv.xaxis.major_label_orientation = np.pi/4
+pv.background_fill_color = "#fafafa"
 
 pv.quad(left=0, bottom=vedges[:-1], top=vedges[1:], right=vhist, color="white", line_color="#3A5785")
 vh1 = pv.quad(left=0, bottom=vedges[:-1], top=vedges[1:], right=vzeros, alpha=0.5, **LINE_ARGS)
 vh2 = pv.quad(left=0, bottom=vedges[:-1], top=vedges[1:], right=vzeros, alpha=0.1, **LINE_ARGS)
 
-pv.min_border_top = 80
-pv.min_border_left = 0
-ph.min_border_top = 10
-ph.min_border_right = 10
-p.min_border_right = 10
-layout = vplot(hplot(p, pv), hplot(ph, Paragraph(width=200)), width=800, height=800)
+layout = column(row(p, pv), row(ph, Spacer(width=200, height=200)))
```

Comment:
It almost works! With the plot axis moved to the right, it aligns the canvas edge instead:
<img width="367" alt="screen shot 2016-06-21 at 9 26 32 pm" src="https://cloud.githubusercontent.com/assets/1078448/16252809/ee5be8a2-37f6-11e6-9fc2-ae30f1a294f0.png">